http-server.sh

```diff
@@ -6,4 +6,5 @@ PUBLIC_PATH="$1"
 shift
 SERVER_OPTS="$@"
 
-./node_modules/.bin/http-server $PUBLIC_PATH -p 8001 -c-1 --cors $SERVER_OPTS
\ No newline at end of file
+echo "starting server with options: $SERVER_OPTS"
+./node_modules/.bin/http-server $PUBLIC_PATH -c-1 $SERVER_OPTS
```

Comment:
Was the removal of --cors intentional here? It's not added back in the ENTRYPOINT above.

Reply:
nope! i will update it back.